### DISCUSSION TITLE
Part2 CH01_10 NACL설정에서 질문있습니다.

### DISCUSSION THREAD

**Discussion by daewonLims** [tedilabs/fastcampus-devops#84] [Q&A] [answered]
인바운드, 아웃바운드 새로 시작하면서 바뀐 규칙 전부 적용해서 EC2 퍼블릭을 만들어서 연결했는데요 
1. 사용자 데이터 입력하는 부분 그대로 했는데 맞는지 모르겠습니다.
    #!/bin/bash
    yum install httpd -y
    service httpd start
3. 퍼블릭 EC2 만들고 결과 화면이 안뜹니다...
4. 아마존도 UI가 바뀌어서 생성하는 부분이 정확히 일치하는지 모르겠습니다.
5. 왜 안되는지 몰라서 네트워크 엑세스 범위 분석을 한번 돌렸는데도 모르겠습니다 ㅠ
![스크린샷 2022-12-14 오후 2 33 40](https://user-images.githubusercontent.com/89571769/207515047-3add2625-ede5-421c-876f-b8c8b1ff90e6.png)

**Comment by daewonLims** [comment, marked as answer]
아... 인스턴스 생성할 때, 아마존 Linux 커널 4.x 버전 꼭 해야하네요...

**Reply by posquit0** [reply]
확인이 늦었네요~ 해결되셔서 다행입니다. :)
